PocketKit/Sources/PocketKit/MyList/ItemsList/ItemSkeletonCell.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 import Textile
 
-class ItemPlaceholderCell: UICollectionViewListCell {
+class ItemSkeletonCell: UICollectionViewListCell {
```

Comment:
renamed as part of convo here: 
https://pocket.slack.com/archives/C01UH57EJKD/p1675801807852719